app/code/ElTrubetskaia/Lesson5/view/frontend/templates/additionalpopupcustomer.phtml

```diff
@@ -0,0 +1,406 @@
+                        <input type="password"
+                               name="modal-password_confirmation"
+                               title="<?php echo $block->escapeHtmlAttr(__('Confirm Password')) ?>"
+                               id="modal-password-confirmation"
+                               class="input-text"
+                               data-validate="{required:true, equalTo:'#modal-password'}"
+                               autocomplete="off">
+                    </div>
+                </div>
+                <?php echo $block->getChildHtml('form_additional_info') ?>
+            </fieldset>
+            <div class="actions-toolbar">
+                <div class="primary">
+                    <button type="submit"
+                            class="action submit primary"
+                            title="<?php echo $block->escapeHtmlAttr(__('Create an Account')) ?>">
+                        <span><?php echo $block->escapeHtml(__('Create an Account')) ?></span>
+                    </button>
+                </div>
+                <div class="secondary">
+                    <a class="action back" href="<?php echo $block->escapeUrl($block->getBackUrl()) ?>">
+                        <span><?php echo $block->escapeHtml(__('Back')) ?></span>
+                    </a>
+                </div>
+            </div>
+        </form>
+    </div>
+</div>
+
+<!--<script type="text/x-magento-init">-->
+<!--    {-->
+<!--        "#registration-for-dealer": {-->
+<!--            "modalRegistrationForm": {-->
+<!--                "config": ""-->
+<!--            }-->
+<!--        }-->
+<!--    }-->
+<!--</script>-->
+
+<script>
+    require([
+        'jquery',
+        'mage/mage'
+    ], function($){
+
+        var dataForm = $('#modal-form-validate');
+        var ignore = <?= /* @noEscape */ $_dob->isEnabled() ? '\'input[id$="full"]\'' : 'null' ?>;
+
+        dataForm.mage('validation', {
+            <?php if ($_dob->isEnabled()): ?>
+            errorPlacement: function(error, element) {
+                if (element.prop('id').search('full') !== -1) {
+                    var dobElement = $(element).parents('.customer-dob'),
+                        errorClass = error.prop('class');
+                    error.insertAfter(element.parent());
+                    dobElement.find('.validate-custom').addClass(errorClass)
+                        .after('<div class="' + errorClass + '"></div>');
+                }
+                else {
+                    error.insertAfter(element);
+                }
+            },
+            ignore: ':hidden:not(' + ignore + ')'
+            <?php else: ?>
+            ignore: ignore ? ':hidden:not(' + ignore + ')' : ':hidden'
+            <?php endif ?>
+        }).find('input:text').attr('autocomplete', 'off');
+
+    });
+</script>
+<?php if ($block->getShowAddressFields()): ?>
+    <script type="text/x-magento-init">
+    {
+        "#country": {
```

Comment:
These selectors haven't need updated! Refer to the conversation in Slack - https://geekhubmagento18.slack.com/archives/CD95VH7NH/p1542745701194300